introduction.md

```diff
@@ -1 +1 @@
-'Hello! I am here to teach people about GitHub.' 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?